src/command.rs

```diff
@@ -28,67 +41,100 @@ pub extern "C" fn wgpu_command_encoder_copy_buffer_to_buffer(
         destination,
         destination_offset,
         size))
+    .expect("Unable to copy buffer to buffer")
+}
+
+#[repr(C)]
+#[derive(Clone)]
+pub struct BufferCopyViewC {
```

Comment:
would it help to slap `repr(C)` on the relevant `wgt` structs?

Reply:
We can do that for now but we will eventually need to make a separate version here to match the [WebGPU headers](https://github.com/webgpu-native/webgpu-headers/blob/c761a96381b73f48d8e0416d0b7b4767e10a3860/webgpu.h#L750)

Reply:
Right, so maybe we should try too hard to re-use it, I agree!